nio.conf

```diff
@@ -13,7 +13,8 @@
 
 # discoverable components that should be disabled from project
 #
-#disabled=SNMPAgent, PubkeeperServer
+#disabled=SNMPAgent
```

Comment:
Is this change intended to be here? We should just disable both components (aka uncomment the line) rather than have two "disabled" lines in the example conf

Reply:
Yes this change was intended, wasn't positive if we needed the SNMP agent so I didn't disable.  Since we're setting up for a nio_lite binary, figured we'd have the PubkeeperServer disabled by default.